platform/platform-commands/pom.xml

```diff
@@ -49,7 +51,8 @@
                         <Export-Package />
                         <Import-Package>
```

Comment:
I think we might be able to get rid of all of the imports. By default if you don't have the <Import-Package> tag it will import *